src/tests/system/tests/test_sss_override.py

```diff
@@ -116,6 +116,61 @@ def test_sss_override__user_attributes(client: Client, provider: GenericProvider
     assert result.home == "/home/o-user1", "User's override name homedir does not match override value!"
 
 
+@pytest.mark.importance("high")
+@pytest.mark.topology([KnownTopology.LDAP, KnownTopology.AD, KnownTopology.Samba])
+def test_sss_overrides__overriding_username_and_attributes_lookup_by_email(client: Client, provider: GenericProvider):
+    """
+    :title: Locally overriding the name and POSIX attributes of a user and lookup with the email address
+    :setup:
+        1. Create POSIX user "user1" with email "email@example.com", email is
+           used because the UPN is not supported in all roles of the framework
+        2. Configure SSSD with "ldap_id_mapping = false" and start SSSD
+        3. Create local override for "user1"
+        4. Restart SSSD, this is necessary to enable local overrides
+    :steps:
+        1. Lookup user by the overridden name, check the uid and gid
+        2. Lookup user by email, check the uid and gid
+    :expectedresults:
+        1. User is found and uid and gid match new values
+        2. User is found and uid and gid match new values
```

Comment:
Please update steps as we are checking lookup with original name and overridden name also
```
    :steps:
        1. Lookup the user by original name
        2. Lookup the user by overridden name
        3. Lookup the user by email address 
```